pilz_teleoperation/test/unittests/tst_ros_message_serializer.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
```

Comment:
Filename should be test_... so pytest will be found (e.g. pycharm does not recognize them otherwise)
see: [Conventions for Python test discovery](https://docs.pytest.org/en/latest/goodpractices.html)